custom_components/octopus_energy/api_client.py

```diff
@@ -328,16 +326,32 @@ def __get_valid_from(self, rate):
   def __get_interval_end(self, item):
     return item["interval_end"]
 
-  def __is_between_local_times(self, rate, target_from_time, target_to_time):
-    """Determines if a current rate is between two times"""
-    local_now = now()
+  def __is_night_rate(self, rate, is_smart_meter):
+    # Normally the economy seven night rate is between 12am and 7am UK time
+    # https://octopus.energy/help-and-faqs/categories/tariffs/eco-seven/)
```

Comment:
Looks like this page has now been replaced by https://octopus.energy/help-and-faqs/articles/what-is-an-economy-7-meter-and-tariff/